Mathlib/GroupTheory/Abelianization.lean

```diff
@@ -307,3 +322,80 @@ instance [Finite (commutatorSet G)] :
   exact Finite.card_pos.ne'
 
 end commutatorRepresentatives
+
+/- # Subgroups containing commutator -/
+
+variable {G}
+
+/-- If G and H have multiplications *
+and φ : G → H is a surjective multiplicative map,
+and if G is commutative, then H is commutative.
+
+Since I only use it with groups,
+I should probably use function.surjective.comm_semigroup
+--/
+theorem Function.Surjective.comm
```

Comment:
This can surely go somewhere else, in a more basic file.

Reply:
Yep.